docker/distelli-manifest.yml

```diff
@@ -1,4 +1,9 @@
 pe-and-platform/puppetdb:
+  PreBuild:
+    - make lint
   Build:
+    - make build
+    - bundle install --path .bundle/gems && bundle exec make test
```

Comment:
I might just put the bundler calls into the Makefile itself, probably as an additional pre-step to `test`, so we can run tests locally just with `make test` and make sure it's "doing the same thing as CI".

Reply:
Updated!